app/templates/releases/canary.hbs

```diff
@@ -19,3 +19,21 @@ npm install --save-dev https://s3.amazonaws.com/builds.emberjs.com{{model.canary
   project=model.emberData
   channel="canary"}}
 </div>
+
+<h2>What is canary?</h2>
+
+<p>
+  "Canary" is whatever code is on the main branch of an Ember project.
```

Comment:
This isn't strictly correct. We do canary builds and then push them to S3, and that's what `ember-source-channel-url` picks up.

Reply:
I changed the following line to "It is not released via npm" which covers this technicality 😄